src/main/play-doc/operation/Install.md

```diff
@@ -724,11 +712,28 @@ Wait until the ConductR instance's health to marked as healthy before proceeding
 
 If more instances are required, scale the application to the desired total and ensure that all of the instances are healthy before proceeding.
 
-### Executor Port Reservation
+Upon startup ConductR will launch its agent process on each of the Mesos slave nodes.
 
-Upon startup ConductR will launch its agent process on each of the Mesos slave nodes. ConductR Agent process will reserve port `2552` for Akka remoting and port range `10000-10999` for exposing endpoints of the bundle deployed within ConductR.
```

Comment:
Should we mention the port ranges reserved by ConductR? I thought this information will be useful during system setup.